### DISCUSSION TITLE
I don't have write access to my github website repository

### DISCUSSION THREAD

**Discussion by Sauroctone** [community/community#40374] [Repositories]
### Select Topic Area

Bug

### Body

Hi,

I have been trying to commit new changes to https://github.com/noelaqueche/noelaqueche.github.io, it had been a while. 
However, it turns out I don't have write access to that repo, and it doesn't appear on my account's repositories list. But the repo's history shows that all commits were done using this account! So I'm very confused, and I need to update the website for job-hunting so this is an issue.

Thanks in advance for the help!

Noé

**Comment by airtower-luna** [comment]
You can see from the URL that the repository belongs to another account. That the commits are connected to your account only means that the author email recorded in the commits is connected to your account, it's entirely unconnected to who authorized a push.

If that other account is yours, too, the only thing to do is to figure out the credentials for that account, and use them.

However, note that having two free accounts [is against GitHub's ToS](https://docs.github.com/en/site-policy/github-terms/github-terms-of-service#b-account-terms), so you might want to [merge them](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-your-personal-account/merging-multiple-personal-accounts) to clean that up.
